ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentSideBar.tsx

```diff
@@ -15,15 +15,44 @@ import {
 } from '@patternfly/react-core';
 
 import useTabs from 'hooks/patternfly/useTabs';
+import { fetchDeployment } from 'services/DeploymentsService';
+import { Deployment } from 'types/deployment.proto';
+import { Model } from '@patternfly/react-topology';
 import DeploymentDetails from './DeploymentDetails';
 import DeploymentNetworkPolicies from './DeploymentNetworkPolicies';
 import DeploymentFlows from './DeploymentFlows';
+import {
+    getListenPorts,
+    getNumExternalFlows,
+    getNumInternalFlows,
+} from '../utils/networkGraphUtils';
+
+type DeploymentSideBarProps = {
+    deploymentId: string;
+    model: Model;
```

Comment:
do we need the whole model here or can we just pass through the `NodeModel` of this deployment instead?